helpers/mathjax.js

```diff
@@ -68,8 +68,9 @@ export class HtmlBlockMathRenderer {
 			elem.querySelectorAll('mspace[linebreak="newline"]').forEach(elm => {
 				elm.setAttribute('style', lineBreakStyle);
 			});
+
 			await window.MathJax.startup.promise;
-			window.MathJax.typeset([elem]);
+			window.MathJax.typesetShadow(elem.getRootNode(), elem);
```

Comment:
Note that this works either way. The `ShadowAdaptor` we have can handle being passed both a shadowRoot and a `document`, so there's no need to do any kind of checking here. Whatever the root node is, we can just pass it along.